apps/mobile/src/features/send/components/recipient/recipient-toggle.tsx

```diff
@@ -28,31 +34,57 @@ export function RecipientToggle({ onPress, value, invalid }: RecipientToggleProp
       }}
     >
       <Box
-        borderColor={invalid ? 'red.border' : 'ink.border-default'}
+        borderColor="ink.border-default"
         borderRadius="sm"
         borderWidth={1}
-        height={64}
         overflow="hidden"
         flexDirection="row"
         justifyContent="space-between"
         alignItems="center"
-        py="4"
+        py="3"
         pl="4"
         pr="2"
       >
         {value ? (
-          <Flag img={<Avatar icon={<UserIcon />} />}>
-            <AddressDisplayer address={value} />
-          </Flag>
+          <AnimatedBox flexDirection="row" entering={enteringAnimation}>
+            <Flag img={<AccountAvatar icon={PersonIcon} />}>
+              <AddressDisplayer address={value} />
+            </Flag>
+          </AnimatedBox>
         ) : (
-          <Text variant="label02" color="ink.text-subdued">
-            {t({
-              id: 'send_form.recipient.toggle.label',
-              message: 'Enter recipient',
-            })}
-          </Text>
+          <>
+            <Text variant="label02" color="ink.text-subdued">
+              {t({
+                id: 'send_form.recipient.toggle.label',
+                message: 'Enter recipient',
+              })}
+            </Text>
+            <IconButton
+              hitSlop={8}
+              label={t({
+                id: 'send-form.recipient.qr_button',
+                message: 'Scan a QR code',
+              })}
+              icon={<QrCodeIcon />}
+              onPress={onQrButtonPress}
+            />
+          </>
         )}
       </Box>
     </Pressable>
   );
 }
+
+function enteringAnimation() {
+  'worklet';
+  return {
+    initialValues: {
+      opacity: 0,
+      //transform: [{ translateY: -3 }],
+    },
+    animations: {
+      opacity: withDelay(80, withTiming(1, { duration: 240 })),
+      //transform: [{ translateY: withTiming(0, { duration: 240 }) }],
```

Comment:
Here too - `//transform`

Reply:
Pressed merge before seeing these. Will remove in a followup, there's few minors items to work on in this flow anyway.